dyn_load.c

```diff
@@ -81,13 +81,6 @@ STATIC GC_has_static_roots_func GC_has_static_roots = 0;
 #   define ELFSIZE ARCH_ELFSIZE
 #endif
 
-#if defined(OPENBSD)
```

Comment:
Why do you remove this block? HAVE_DL_ITERATE_PHDR should not be defined for latest OpenBSD? Or it is always defined by configure?

Reply:
It is always defined by configure now.

Reply:
Got it. I prefer to keep this code for now. Because 4 build systems are supported (for OpenBSD), but HAVE_DL_ITERATE_PHDR is provided only in the 1st two of them:
* configure
* CMake
* Makefile.direct
* none (e.g. gcc -I include extra/gc.c)

.

Reply:
Okay. This was more of a cleanup of what I thought was unneeded code. Keeping it is fine as well.